opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreCommunityRespondersRegisterActivity.java

```diff
@@ -100,7 +101,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
             if (repo.getRespondersCount() > 7) {
                 Toast.makeText(this, getString(R.string.add_responder_max_message), Toast.LENGTH_LONG).show();
             } else {
-                JSONObject form = FormUtils.getFormUtils().getFormJson(CoreConstants.JSON_FORM.COMMUNITY_RESPONDER_REGISTRATION_FORM);
+                JSONObject form = null;
+                try {
+                    form = FormUtils.getFormUtils().getFormJson(CoreConstants.JSON_FORM.COMMUNITY_RESPONDER_REGISTRATION_FORM);
+                } catch (IOException e) {
+                    Timber.e(e);
+                }
                 startFormActivity(form);
```

Comment:
This would result on an NPE, we should execute all the code referencing the form within the try catch